active/AnimatedIcon/AnimatedIconSpec.md

```diff
@@ -0,0 +1,417 @@
+        < VisualState x: Name =”Normal”> 
+        </ VisualState >       
+        < VisualState x: Name =”PointerOver”>           
+            < VisualState.Setters >            
+                < Setter Target = "Icon.(AnimatedIcon.StateProperty)" Value = "Hover" />               
+            </ VisualState.Setters >               
+        </ VisualState >              
+        < VisualState x: Name =”Pressed”>                                   
+        </ VisualState >                    
+        </ VisualStateGroup >                    
+    </ VisualStateManager.VisualStateGroups >
+    < ContentPresenter
+    x: Name = "ContentPresenter"
+    Background = "{TemplateBinding Background}"
+    BorderBrush = "{TemplateBinding BorderBrush}"
+    BorderThickness = "{TemplateBinding BorderThickness}"
+    Content = "{TemplateBinding Content}"
+    ContentTemplate = "{TemplateBinding ContentTemplate}"
+    ContentTransitions = "{TemplateBinding ContentTransitions}"
+    Padding = "{TemplateBinding Padding}"
+    HorizontalContentAlignment = "{TemplateBinding HorizontalContentAlignment}"
+    VerticalContentAlignment = "{TemplateBinding VerticalContentAlignment}"
+    AutomationProperties.AccessibilityView = "Raw" />
+< Grid x: Name = "Icon" HorizontalAlignment = "Right" Content = "{TemplateBinding Icon}/>
+```
+
+After the template is updated, you can add your animated icon just like any other icon to your NavigationViewItem: 
+
+XAML
+
+```xml
+< muxc:NavigationView.MenuItems >
+    < muxc:NavigationViewItem Content = "Menu Item1" Tag="SamplePage" x:Name = "SamplePageItem" >
+        < muxc:NavigationViewItem.Icon >
+            < controls:AnimatedIcon x:Name = "PlayAnimatedIcon" >
+                < controls:AnimatedIcon.Source >
+                    < local:PlayIcon />
+                </ controls:AnimatedIcon.Source>
+            </ controls:AnimatedIcon >
+    </ muxc:NavigationViewItem.Icon >
+</ muxc:NavigationView.MenuItems >
+```
+
+# Add an animated icon to a button 
+
+If you would like to add an animated icon to a control that does not have the default template updated, you will need to update the template to include the state changes needed to play the animation segments. 
+
+Here is the XAML for the changes you would need to make to add an AnimatedIcon to a button. This includes the changes you will need to make to the state transitions for Normal, PointerOver, Pressed, and Disabled. This will update the button template to add the state transitons needed for AnimatedIcon to map the animation segment in the PlayIcon file to the correct visual state.
+
+This example also shows how to add the fallback static icon that will be used if the animated icon cannot be played. In this example we are using a SymbolIcon glyph. 
+
+XAML
+```xml
+<Button Click="Button_Click"
+        Background="LightGray"
+        Height="100" Widht="80">
+        <AnimatedIcon Source="PlayIcon" FallBackSource="{SymbolIconSource Symbol="Play"}"/>
```

Comment:
Does {SymbolIconSource} work? I've been wanting to add support for that syntax, but Xaml doesn't have it today.